CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # App Center SDK for iOS and macOS Change Log
 
+## Version 1.12.0 (Under active development)
```

Comment:
Let's use `1.11.1` for now and bump to `1.12.0` when we need to.

Reply:
We actually know that the next would be 1.12.0 but I agree this one since there is no features already added.